### ISSUE TITLE
Remove columns from table

### ISSUE THREAD

**Issue by tischi** [hci-unihd/batchlib#52]
@imagirom @constantinpape 

There are also some columns in the table that might be too much information for "general usage", e.g. 
![image](https://user-images.githubusercontent.com/2157566/80476872-4caa4080-894b-11ea-93a6-8b1269d509ea.png)

However, maybe it is fine to leave everything in the table and then the consumer could decide not to show all of them, e.g. the PlateViewer would be default only show a minimal subset.

What do you think?

**Comment by constantinpape** [comment]
Yes, I agree. It would be good to enable not showing all columns by default.
However, I am not quite sure what's the best way to enable this for the csv table format.

For the hdf5 tables it would be easy, we would just include a new dataset `visible` that would be true or false for each column.

**Comment by tischi** [comment]
Some ideas:

1. We could do it like in the PlatyBrowser, with one default.csv and then more associated tables.
2. We just save the tables as hdf5
3. Each consumer takes the liberty to decide on its own which ones to show

**Comment by constantinpape** [comment]
>     2\. We just save the tables as hdf5

I personally like this option the most, then we have the same table set-up throughout.
For the analysis, we would then just add an H5 file `<plate_name>_analysis.h5`.
This file can then contain multiple table and we have a in-file link `tables/default` or so that points to the table that is loaded automatically.

We should still support csv tables as well though, because it is more convenient to work with for some more custom things that don't use batchlib to write the tables in our hdf5 table format. (But in this case we don't need to support hiding columns).

**Comment by tischi** [comment]
I am also interesting in using hdf5 for this. 
At some point it may also be a database.
Maybe it does not hurt to save it three times: csv/h5/db.

**Comment by constantinpape** [comment]
> I am also interested in using hdf5 for this.
> At some point it may also be a database.
> Maybe it does not hurt to save it three times: csv/h5/db.

Agreed to all.
Regarding db we need to see how to actually do this once you and @wolny have explored this further.

In any case, I will update the pipeline at some point later to write out csv and h5 once Roman and me have integrated our current changes.

**Comment by tischi** [comment]
Regarding the `Show by default` in the hdf5: I am becoming a fan of using normal arrays where possible and avoid using attributes, because they are kind of hidden and more annoying to parse. In fact they just adds a layer of complexity that is maybe not necessary and might make things less compatible, also having a database and n5 in mind. 

So why not just have a 
```
Boolean[ ] default = {true, false, false, ...}
```

**Comment by constantinpape** [comment]
> So why not just have a
> 
> ```
> Boolean[ ] default = {true, false, false, ...}
> ```

Yes, that's exactly what I will do. Now that we have a sub-structure for the tables already, there's no need to resort to attributes for this.
A table would then look like this:
```
table/
  cells
  columns
  visble
```

**Comment by constantinpape** [comment]
This is implemented now in #51.
@tischi let me know if you find any bugs and reopen if necessary.
